odoo/addons/base/tests/test_res_partner.py

```diff
@@ -757,26 +757,26 @@ def test_100_res_partner_recursion(self):
     # split 101, 102, 103 tests to force SQL rollback between them
 
     def test_101_res_partner_recursion(self):
-        with self.assertRaises(ValidationError):
+        with self.assertRaises(UserError):
```

Comment:
I think it should still be a `ValidationError`. Probably the function `_parent_store_update` should raise it instead of the `UserError`.

Reply:
Mhhh, yes you are right, I will change that. Thx